src/use-cases/addproject.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, describe, it, beforeEach } from 'vitest'
+
+import { CreateProjectUseCase } from './addProject'
+import { InMemoryProjectRepository } from '../repositories/in-memory-db/inMemoryProjectRepository'
+import { InMemoryUserRepository } from '../repositories/in-memory-db/inMemoryUserRepository'
+
+let projectRepository: InMemoryProjectRepository
+let userRepository: InMemoryUserRepository
+let sut: CreateProjectUseCase
```

Comment:
Could you please ensure that the new tests adhere to the standards of the existing ones? If you prefer to refer to them as 'system under test,' that's a valid option, but you'll need to rename the other tests accordingly.